commons/src/main/java/gov/cms/qpp/conversion/util/StringHelper.java

```diff
@@ -16,6 +17,8 @@
 	 * @return A joined {@link String}.
 	 */
 	public static String join(Iterable<String> iterable, String separator, String conjunction) {
+		separator = separator + " ";
```

Comment:
Doesn't java already include a `String.join` that fits the bill?

Reply:
No it doesn't.  :(  Not one that understands a conjunction between the second-to-last and last element.

Reply:
Ah, gotcha